source/drivers/csharp.txt

```diff
@@ -20,11 +20,9 @@ Introduction
 The official MongoDB C#/.NET Driver provides asynchronous interaction
 with MongoDB.
 
-- :csharp-docs-latest:`Getting Started </getting_started/>`
+- :csharp-docs-latest:`Usage Guide </getting_started/>`
```

Comment:
Should the link name be changed? The section it links to is called Getting Started, so it could be confusing to jump context without an explanation.

Reply:
It's meant to follow the structure of the C++ page: https://docs-mongodbcom-staging.corp.mongodb.com/ecosystem/pegasus/DOCS-12433/drivers/cxx.html
which also contains a "Usage Guide" that just points to a general landing page. Take a look at the comments on the JIRA ticket because I'm mostly following those.

Reply:
changing